include/Morimura/mrUtil.h

```diff
@@ -8,23 +8,46 @@
 struct J2DPane;
 
 namespace Morimura {
+
 struct TOffsetMsgSet {
 	TOffsetMsgSet(u64*, u64, int);
+	TOffsetMsgSet(u64*, u64, int, u64*, int*);
 
 	u64 getMsgID(int);
 
-	u64* _00;    // _00
-	int* _04;    // _04
-	u64 m_msgID; // _08
-	int _0C;     // _0C
-	int m_size;  // _10
+	inline void calcOffset(int& offset, int i)
+	{
+		int calc  = pow(10.0f, i);
+		int calc2 = offset / calc;
+		_04[i]    = offset / calc;
```

Comment:
Could _04 have a name based on this? Haven't looked into this code specifically but perhaps it can be derived